onebusaway-android/src/main/java/org/onebusaway/android/ui/HomeActivity.java

```diff
@@ -976,10 +976,10 @@ private void goToContactEmail(Context ctxt) {
     private void goToSendFeedBack() {
         if (mFocusedStop != null) {
             ReportActivity.start(this, mFocusedStopId, mFocusedStop.getName(),
-                    mFocusedStop.getLatitude(), mFocusedStop.getLongitude());
+                    mFocusedStop.getLatitude(), mFocusedStop.getLongitude(), mGoogleApiClient);
         } else {
             Location loc = Application.getLastKnownLocation(this, mGoogleApiClient);
```

Comment:
This `loc` can be `null` - can you please update this to protect against an NPE?  I believe this means you'll need to allow for the situation where the lat and long aren't available within `ReportActivity`.  This would also apply to `HomeActivity.goToSendFeedBack()`.